### PR TITLE
chore: try to fix screenshot

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
加了两个配置 `onlyViewport` 和 `splitTheme`，分别用于只截取视区和单独截图三个预设主题。以后有问题的 demo 截图都可以用这两个配置来稳定。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 9a7fafd</samp>

Refactored image snapshot tests to use dynamic viewport height and fixed viewport width for better test results. Updated `tests/shared/imageTest.tsx` file.

### <samp>🤖 Generated by Copilot at 9a7fafd</samp>

* Improve image snapshot tests by setting viewport width to 1920 and height to 1, and waiting for network idle before taking screenshot ([link](https://github.com/ant-design/ant-design/pull/44817/files?diff=unified&w=0#diff-34d634572b80c540ebecd7780004e6031fbb9f004387e3df7302e346940ffae4L40-R44), [link](https://github.com/ant-design/ant-design/pull/44817/files?diff=unified&w=0#diff-34d634572b80c540ebecd7780004e6031fbb9f004387e3df7302e346940ffae4L58-L60), [link](https://github.com/ant-design/ant-design/pull/44817/files?diff=unified&w=0#diff-34d634572b80c540ebecd7780004e6031fbb9f004387e3df7302e346940ffae4L78-R84))
* Remove unnecessary `div` element with id `end-of-screen` from `tests/shared/imageTest.tsx` ([link](https://github.com/ant-design/ant-design/pull/44817/files?diff=unified&w=0#diff-34d634572b80c540ebecd7780004e6031fbb9f004387e3df7302e346940ffae4L58-L60))
* Adjust viewport height dynamically to match scroll height of rendered page using `page.evaluate` and `page.setViewport` in `tests/shared/imageTest.tsx` ([link](https://github.com/ant-design/ant-design/pull/44817/files?diff=unified&w=0#diff-34d634572b80c540ebecd7780004e6031fbb9f004387e3df7302e346940ffae4L78-R84))

### CODE REVIEW

**Comment by MadCcc** [comment]
这个改动似乎把时间拖得比较久

**Review by kiner-tang — APPROVED** [review]

File: components/layout/__tests__/image.test.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import { imageDemoTest } from '../../../tests/shared/imageTest';
 
 describe('Layout image', () => {
-  imageDemoTest('layout', { skip: ['fixed-sider.tsx'] });
+  imageDemoTest('layout', {
+    skip: ['fixed-sider.tsx'],
+    onlyViewPort: ['side.tsx'],
```
Comment (afc163) [inline components/layout/__tests__/image.test.ts:6]: onlyViewPort 是啥意思。

File: components/select/__tests__/image.test.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import { imageDemoTest } from '../../../tests/shared/imageTest';
 
 describe('Select image', () => {
-  imageDemoTest('select', { skip: ['basic.tsx'] });
+  imageDemoTest('select', {
+    skip: ['basic.tsx'],
```
Comment (afc163) [inline components/select/__tests__/image.test.ts:5]: loading 类的也可以把转圈动画覆盖掉后，都开起来。

File: components/layout/__tests__/image.test.ts (hunk 1 above)
Comment (MadCcc) [reply components/layout/__tests__/image.test.ts:6]: 只截取 viewport 的部分。有些 demo 样式里包含 vh，现在动态设置 viewport 会让这些截图不稳定不可读，所以做一些限制。

File: components/select/__tests__/image.test.ts (hunk 2 above)
Comment (MadCcc) [reply components/select/__tests__/image.test.ts:5]: 嗯，现在截图里把动画都关掉了，这里可以去掉了

**Pull request merged**